backend/app/services/qualtrics_test_data.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'faker'
```

Comment:
is faker used here?

Reply:
yes, it's the calls to `Faker:Lorem` below